fms/modules/attention.py

```diff
@@ -83,28 +87,25 @@ def __init__(
 
     def reset_params(self, gain=1):
         # Ensure softmax inputs are standard normal
-        for layer in ["query", "key"]:
-            nn.init.trunc_normal_(
-                getattr(self, layer).weight, mean=0.0, std=self.emb_dim**-0.5
-            )
+        self.qkv_fused.weight.data[
+            : self.emb_kq_per_head * (self.nheads + self.kvheads)
+        ].normal_(0, self.emb_dim**-0.5)
```

Comment:
just fyi this is going to end up conflicting with https://github.com/foundation-model-stack/foundation-model-stack/pull/177

Reply:
eventually... https://github.com/foundation-model-stack/foundation-model-stack/pull/186